src/xdocs/config_whitespace.xml

```diff
@@ -2018,15 +2018,27 @@ public @interface Beta {} // empty annotation type
         <p>
         may optionally be exempted from the policy using the <code>
         allowEmptyMethods</code>, <code>allowEmptyConstructors
-        </code>, <code>allowEmptyTypes</code>, <code>allowEmptyLoops</code> and
-        <code>allowEmptyLambdas</code> properties.
+        </code>, <code>allowEmptyTypes</code>, <code>allowEmptyLoops</code>
+        <code>allowEmptyLambdas</code> and <code>allowEmptyCatches</code>
+        properties.
         </p>
         <p>This check does not flag as violation double brace initialization like:</p>
         <pre><code>
 new Properties() {{
     setProperty("key", "value");
 }};
         </code></pre>
+        <p>Parameter allowEmptyCatches allows to suppress violations when token
+        list contains SLIST to check if beginning of block is surrounded by
```

Comment:
I added this because of @romani suggestion:

> > Removing catch token from list of tokens does not help, because problem with SLIST persist. Thats why EmptyCatchBlock has nothing to do with this case.
> 
> please add this to xdoc to deliver that nuance to user. It will be useful.

In my view parameter is so self explanatory it does not need an example, and issue with SLIST is a subset of general problem with empty catch blocks in WhitespaceAround check.